### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this pull, we(me and my coding partner) have worked on:
- Adding contactUs as footer 
- Designing form according to Figma design
- Adding Form validation
- Fixed textarea placeholder
- Fixed button placement
Screen Shot 2022-06-01 at 12 18 14 PM

Kindly review it and share your feedback with me

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,203 +1,294 @@
...
-</body>
+        <input
+          name="email"
+          id="user-email"
+          type="email"
+          placeholder="name@gmail.com"
+          required
+        />
 
+        <textarea
```
Comment (Ismailco) [inline index.html:282]: - [ ] The form will send even if the user didn't add a text message, please fix this you can remove the text inside the ```textarea``` (Write your message here) and replace it using a placeholder
- [ ] Please try to use a placeholder for the ```textarea```, it's a requirement

File: index.html [hunk 2]
```diff
@@ -1,203 +1,294 @@
...
+          name="message"
+          maxlength="500"
+          required
+        >
+        Write your message here
+        </textarea>
+        <button type="submit">Get in Touch</button>
```
Comment (Ismailco) [inline index.html:289]: - [ ] The submit button should be aligned to the left of the form, please pay more attention to details and try to creat the Figma design as close as possible

|Your design|Figma design|
|-|-|
|Screenshot_20220531_212058|Screenshot_20220531_212428|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi Team,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @shahira-sadat , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**